mkpydeps

```diff
@@ -13,31 +13,62 @@ spits out Homebrew resource stanzas.
 import os, sys, json, urllib2
 from hashlib import sha1
 import tl.eggdeps.graph
+import pip
+
+pkg = sys.argv[1]
 
 graph = tl.eggdeps.graph.Graph()
-graph.from_specifications([sys.argv[1]])
-deps = set(graph.keys()) - set([sys.argv[1]])
+graph.from_specifications([pkg])
+deps = set(graph.keys()) - set([pkg])
+installed_packages = pip.get_installed_distributions()
+versions = dict((el.key,el.version) for el in installed_packages)
 
-for dep in deps:
-    f = urllib2.urlopen("http://pypi.python.org/pypi/{}/json".format(dep))
+def package_info(pkg, version):
+    f = urllib2.urlopen("http://pypi.python.org/pypi/{}/{}/json".format(pkg,
+        versions[pkg]))
     j = json.load(f)
     f.close()
-    print '  resource "{}" do'.format(dep)
     for i in j['urls']:
         if i['packagetype'] == 'sdist':
             url = i['url']
-            print '    url "{}"'.format(url)
             f = urllib2.urlopen(url)
             checksum = sha1(f.read()).hexdigest()
-            print '    sha1 "{}"'.format(checksum)
+            f.close()
             break
-    print '  end'
+    return {
+            'home_page': j['info']['home_page'],
```

Comment:
Does PyPI require that this is always defined? j['info'].get('home_page', '')` is a little safer.